### PR TITLE
Add build page

### PR DESCRIPTION
**Pull request by petey**
Adds a build page to display build details, step information, and logs. Provides colored log output for steps that have started after clicking on the step.

TODO:
- [x] Eslint instead of jshint (ember-cli-eslint hasn't updated to eslint 3.x, yet, so we can't import the screwdriver rules directly)
- [x] Use istanbul for code coverage 
- [x] More documentation in components

Will do more acceptance tests in later PR.

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /* jshint node: true */
 
-module.exports = function(environment) {
-  var ENV = {
+module.exports = (environment) => {
```
Comment (nkatzman) [inline config/environment.js]: does this get called with the `NODE_ENV` variable?

Comment (petey) [inline config/environment.js]: environment contains the name of the current environment (development,production or test).

https://guides.emberjs.com/v2.7.0/configuring-ember/configuring-your-app/

File: tests/unit/build/adapter-test.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+// let passedVerb;
+// let passedHash;
+
+/**
+ * Mock ajax for test
+ * @method ajaxResponse
+ * @param  {Mixed}  value   Mocked api response
+ * @return {Promise}
+ */
+// function ajaxResponse(value) {
```
Comment (tkyi) [inline tests/unit/build/adapter-test.js]: is this function still needed?

File: tests/unit/build/model-test.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+moduleForModel('build', 'Unit | Model | build', {
+  // Specify the other units that are required for this test.
+  needs: []
+});
+
+test('it exists', function (assert) {
+  let model = this.subject();
+
+  // let store = this.store();
```
Comment (tkyi) [inline tests/unit/build/model-test.js]: do you need this variable anymore?

File: app/components/sd-build-step/component.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  status: Ember.computed('step.code', 'step.startTime', {
+    get() {
+      const code = this.get('step.code');
+      const startTime = this.get('step.startTime');
+
+      if (!startTime) {
+        return 'queued';
+      }
+
+      if (code === undefined && startTime) {
```
Comment (tkyi) [inline app/components/sd-build-step/component.js]: not sure if it makes a difference, but you could also use `Ember.none(code)`
http://emberjs.com/api/classes/Ember.computed.html#method_none

**Comment by nkatzman** [comment]
You've answered all the questions I have with my limited UI knowledge. 👍

**Comment by tkyi** [comment]
👍 LGTM
giphy-1

File: app/components/sd-build-step/component.js (hunk 4 above)
Comment (petey) [inline app/components/sd-build-step/component.js]: Making a computed property specifically for `code === undefined` might be overkill. I'm not sure how `Ember.computed.none()` even works in a scalar context.

**Pull request merged**